src/components/List.js

```diff
@@ -12,14 +22,18 @@ const List = () => {
           return isLoading ? (
             <p>loading...</p>
           ) : (
-            <div>
-              <ul style={{ listStyleType: 'none' }}>
-                {data.map(item => (
-                  <li key={item.id}>
-                    {item.name} - next purchase in {item.nextPurchase} days
-                  </li>
-                ))}
-              </ul>
+            <div className="List">
```

Comment:
Not a blocker by any means, you've totally met the Acceptance Criteria, but I might **suggest** adding just a bit more detail here for the user to let them know what they're seeing. This is what one of the last cohorts put, but it's totally up to you. Kinda put yourself in the shoes of a user and think about what you'd wanna see here to guide you through using the app. 🤔

Reply:
Absolutely. Thanks for this reminder.